Configuration/AlCa/python/autoAlca.py

```diff
@@ -1,6 +1,6 @@
 AlCaRecoMatrix = {'AlCaLumiPixels' : 'LumiPixels',
                   'Charmonium'     : 'TkAlJpsiMuMu',
-                  'Commissioning'  : 'HcalCalIsoTrk',
+                  'Commissioning'  : 'HcalCalIsoTrk+HcalCalIsolatedBunchSelector',
                   'Cosmics'        : 'TkAlCosmics0T+MuAlGlobalCosmics+HcalCalHOCosmics+DtCalibCosmics',
```

Comment:
@bsunanda we miss the `HcalCalIsolatedBunchFilter` on the `JetHT` PD as in [81x version of autoAlCa](https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Configuration/AlCa/python/autoAlca.py)